docker/dev/centos/README.md

```diff
@@ -0,0 +1,46 @@
+# How to use this Dockerfile
+
+This image is intended for development of Orion Context Broker. This will create an environment on which Orion can compile and be run based on CentOS 6.
+
+If you are only interested in running Orion Context Broker in a container this image is not what you are looking for. 
+
+## Build the image
+
+You only need to do this once.
+
+    docker build -t orion-dev:centos6 .
+    
+## Run the container
+
+This assumes that there is another docker running with name $mongodb$. If you don't, type this:
+
+    docker run -d --name mongodb mongo
+
+The following line will run the container exposing port 1026, give it a name -in this case $orionDev$-, link it to the mongodb docker, and present a bash prompt. Keep in mind that /path/to/orion/source is the path to your Orion Context Broker code in the host, not the container. 
+
+    docker run -p 1026:1026 --name orionDev -v ~/src/fiware-orion:/root/src --link mongodb:mongodb -t -i orion-dev:centos6 /bin/bash
+
+## Compile Orion
+
+Before you compile you must disable proxyCoap (not supported in this docker image):
+
+    sed -i '/proxyCoap/s/^/#/' /root/src/CMakeLists.txt    
+
+Now you can compile Orion Context Broker:
+
+    cd /root/src
+    make
+    make install
+
+## Run Orion
+
+To run orion you can do it interactively:
+
+    contextBroker -fg -t 0-255
+
+Or as on the background
+    
+    contextBroker -t 0-255
+
+Take a look a the [documentation](http://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_Installation_and_Administration_Guide#Command_line_options) for all command-line options.
```

Comment:
Documenation is no longer in the wiki. Please, have a look to the docs/admin directory to find the right .md to cite (if you don't find it, please tell me about).